core-war/src/main/webapp/util/javaScript/vuejs/components/comments/silverpeas-comments-templates.jsp

```diff
@@ -120,18 +120,20 @@ other users. The update opens a popup to modify the text of the comment.
         </silverpeas-form-pane>
       </silverpeas-popin>
 
-      <silverpeas-fade-transition-group id="list-box"
-                                        v-if="comments.length"
-                                        tag="div">
-        <silverpeas-comment v-for="comment in comments"
-                            v-bind:key="comment.id"
-                            v-bind:current-user="user"
-                            v-bind:comment="comment"
-                            v-bind:readonly="false"
-                            v-on:comment-update="updateComment"
-                            v-on:comment-deletion="deleteComment">
-        </silverpeas-comment>
-      </silverpeas-fade-transition-group>
+      <silverpeas-fade-transition>
+        <div id="list-box" v-if="comments.length">
+          <silverpeas-fade-transition-group>
+            <silverpeas-comment v-for="comment in comments"
+                                v-bind:key="comment.id"
+                                v-bind:current-user="user"
+                                v-bind:comment="comment"
+                                v-bind:readonly="false"
+                                v-on:comment-update="updateComment"
+                                v-on:comment-deletion="deleteComment">
+            </silverpeas-comment>
+          </silverpeas-fade-transition-group>
+        </div>
+      </silverpeas-fade-transition>
```

Comment:
What is the difference between silverpeas-fade-transition and silverpeas-fade-groupe-transition? Why using here the first one as the upper element of the second?

Reply:
`silverpeas-fade-transition` MUST encapsulate a single root node to be functional.
Whereas `silverpeas-fade-groupe-transition` MUST be used when several root nodes are handled, such as the `<li>` nodes of an `<ul>` or `<ol>` list. Or such as several direct `<div>` nodes included into an other `<div>`.

Reply:
As I said, because of the "v-if" condition.
Call me if you want more explanation.

Reply:
OK.
I did some more testing by removing `<silverpeas-fade-transition>` and it now works fine.
Maybe I did some migrations around the fade functionality after migrating the comment functionality.
It seems that with VueJS 3.x, the fade transition group is now better handled.
So I updated the PR by removing `<silverpeas-fade-transition>` on the group one.